### ISSUE TITLE
No Error Message For Invalid Characters

### ISSUE THREAD

**Issue by packagedjack** [rstudio/shiny#2393]
While populating a data-frame with dbGetQuery from DBI, a character stops the app from functioning.

Specifically this one select ascii('’') ( this is ran from SQL Server), the ASCII value is 146. I'm not certain if this is a DBI issue or shiny, but the fact that there is no error messaging at all leads me to believe that his is an un handled exception in shiny.

**Comment by alandipert** [comment]
Thank you for the report. When you say the app stops functioning, do you mean it grays out?

Also, how are you running the application? (Connect, Shiny Server, locally)

Depending on how you're running the application, and how it fails, error logs are created on the server that could illuminate the problem.

A standalone way to reproduce the problem would be ideal, but I understand that because of the involvement of DBI that would be difficult or impossible. So, thanks in advance for providing any additional information you can.

**Comment by packagedjack** [comment]
Yes it grays out. I'm running it through Rstudio on my local machine. We
have a server but this is for a new application before we deploy, so there
are no logs quite yet. I will try to attach an example of the issue through
a dummy app, shouldn't be too tough.
